previousnext/agov
# [#28931] Added picture and focal_point modules

To test:
* Enable breakpoint, picture and focal_point modules
* In theme setup breakpoints, eg (dcomms_theme.info) NOTE highest to lowest when using min-width, it uses the first one that satisfies rule:
```
breakpoints[normal] = (min-width: 720px)
breakpoints[mobile] = (min-width: 0px)
```
* Check breakpoints show in UI (eg. admin/config/media/breakpoints/groups/dcomms_theme). May require cache clear
* Setup image styles wish to use for each breakpoint. This is where focal_point comes in, can use its effects. For example can crop out portrait image from a landscape image.
* Setup Picture mappings (admin/config/media/picture) to select image style for each breakpoint
* For the field display settings choose 'Picture' formatter and select the mapping wish to use

So in the context of the dcomms story, I setup a new view mode for Quick Item called News Landscape that uses the Picture formatter with News Landscape mapping.

![](https://www.dropbox.com/s/lkcczwpppo3yxkk/Screenshot%202015-01-30%2014.56.45.png?dl=1)
